harness/nbjunit/src/org/netbeans/junit/MethodOrder.java

```diff
@@ -43,6 +43,9 @@ static void initialize() {
         String orderS = findOrder();
         if (!"natural".equals(orderS)) { // NOI18N
             try {
+                // TODO: ClassLoader fields can not be accessed via reflection on JDK 12+
+                // see jdk.internal.reflect.Reflection#fieldFilterMap
+                // this won't work
                 Field classesF = ClassLoader.class.getDeclaredField("classes"); // NOI18N
```

Comment:
luckily nothing seems to be using test case ordering: https://github.com/search?q=repo%3Aapache%2Fnetbeans%20NbTestCase.order&type=code

so I haven't bothered to fix it.